NOTICE.txt

```diff
@@ -1,6 +1,6 @@
 StarRocks
 
-Copyright 2021 StarRocks Limited
+Copyright 2022 StarRocks Limited
```

Comment:
Should be 2021 - present other than 2022

Reply:
OK